tests/ci/unit_tests_staging.yml

```diff
@@ -0,0 +1,86 @@
+# unit_tests_staging.yml
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+#
+# use variable group name
+variables:
+- group: AzureKeyVaultSecrets
+
+#trigger:
+#- azure-pipelines-bz
+#  - master
+
+#pr:
+#- staging 
+  
+pool:
+ vmImage: 'ubuntu-16.04'
+
+steps:
+
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.6'
+    architecture: 'x64'
+  displayName: 'Use Python 3.6'
+
+
+- script: | 
+   az login --service-principal -u $(ClientID) -p $(ClientSecret) --tenant $(TenantID)
+   
+  displayName: 'Login to Azure'
+
+- script: |
+   sed -i 's#"subscription_id": "<>"#"subscription_id": "$(SubscriptionID)"#g' ./tests/ci/config.json
+   echo my subscription is $(SubscriptionID)
+   cat ./tests/ci/config.json
+  displayName: 'replace subscription value'
+
+- script: 
+   sed -i 's#"tests/unit_or_smoke_int"#"tests/unit"#g' ./tests/ci/runpytest.py
+  displayName: 'replace unit or smoke or int'
+
+- script:
+    sed -i 's#"not notebooks and not spark and not gpu"#"not notebooks and not spark and not gpu"#g' ./tests/ci/runpytest.py
+  displayName: 'notebooks and spark and gpu'
+
+- bash: |
+    echo "##vso[task.prependpath]/data/anaconda/bin"
+  displayName: Add Conda to PATH
+
+- script: 'pip install azureml-sdk'
+  displayName: 'install azureml-sdk'
+  continueOnError: true
+  
+- script: 
+     python scripts/generate_conda_file.py
+  displayName: ' generate_conda_file.py'

```

Comment:
parameters for this will need to change based on using spark, gpu or base python

Reply:
agree. The pipeline will be different for spark, gpu or base python. I can pass in a par for that based on the pipelines once we get this first one tested but don't want to hold this up for that.